registrations/tests/test_signup_put.py

```diff
@@ -110,6 +111,39 @@ def test__created_regular_user_can_update_signup(
     assert db_signup.last_modified_by_id == user.id
 
 
+@freeze_time("2023-03-14 03:30:00+02:00")
+@pytest.mark.django_db
+def test__created_regular_user_cannot_update_signup_presence_status(
+    user_api_client, registration, signup, user
+):
+    signup.created_by = user
+    signup.save(update_fields=["created_by"])
+
+    user.get_default_organization().regular_users.add(user)
+    user.get_default_organization().admin_users.remove(user)
+
+    new_signup_name = "Edited name"
+
+    db_signup = SignUp.objects.get(pk=signup.id)
+    assert db_signup.first_name != new_signup_name
+    assert db_signup.last_modified_by_id is None
+    assert db_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
+
+    signup_data = {
+        "registration": registration.id,
+        "first_name": new_signup_name,
+        "date_of_birth": "2015-01-01",
+        "presence_status": SignUp.PresenceStatus.PRESENT,
+    }
+
+    response = update_signup(user_api_client, signup.id, signup_data)
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+    db_signup = SignUp.objects.get(pk=signup.id)
+    assert db_signup.first_name != new_signup_name
+    assert db_signup.last_modified_by_id is None
+    assert db_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
```

Comment:
Maybe you could also test that authenticated user without organization can update `presence_status` of signup/signup group if created by the user?